pkg/issuer/acme/acme.go

```diff
@@ -34,15 +34,16 @@ import (
 // certificates from any ACME server. It supports DNS01 and HTTP01 challenge
 // mechanisms.
 type Acme struct {
-	*controller.Context
```

Comment:
Decided to break out references to things in the controller context here into dedicated fields to more closely match other controllers & make testing not require a full Context :)